common.mk

```diff
@@ -301,6 +308,10 @@ filelist-tee-common: optee-client xtest helloworld
 			"$(HELLOWORLD_PATH)/ta/8aaaf200-2450-11e4-abe2-0002a5d5c51b.ta" \
 			"444 0 0" 					>> $(fl); \
 	fi
+	@if [ "$(QEMU_USERNET_ENABLE)" = "y" ]; then \
```

Comment:
That could probably be done unconditionally provided that `/etc/init.d/udhcpc` properly checks if `eth0` is available (which you do in your other PR, AFAICT).